Doc/library/cmath.rst

```diff
@@ -340,6 +340,8 @@ Constants
    A floating-point "not a number" (NaN) value.  Equivalent to
    ``float('nan')``.
 
+   See also :data:`math.nan`.
+
```

Comment:
cmath.nan is more equivalent to math.nan than float('nan') in being a fixed data object, with a fixed id, rather than a function which can return different objects with different ids when called multiple times.
```suggestion
    :data:`math.nan`.

```